strax/context.py

```diff
@@ -1988,6 +1988,22 @@ def is_stored(self, run_id, target, **kwargs):
             if self._is_stored_in_sf(run_id, target, sf):
                 return True
         # None of the frontends has the data
+
+        # Before returning False, check if the data can be made trivially
+        plugin = self._plugin_class_registry[target]
+        save_when = plugin.save_when
+
+        # Mutli-target plugins provide a save_when per target
+        if isinstance(save_when, immutabledict):
+            save_when = save_when[target]
+
+        if save_when < 3:  # 3 is SaveWhen.ALWAYS
+            warnings.warn(
```

Comment:
Hej @cfuselli you could add within this if-statement something like:
```python
try:
    components = st.get_components('058518', target)
    targets_plugin_made_from = components.loaders.keys()
    # Warn user:
    print('Can be made from: ', targets_plugin_made_from)
except strax.DataNotAvailable:
    # Warn that data cannot be made
    print('cannot be made')
```

Reply:
Nice but I think it goes way beyond the scope of is_stored, no? 
The warning is just intended to inform user of the existence of plugins that are not stored, so that they won't use these plugins to check if something is stored.

Reply:
Hey @WenzDaniel. Shal I still mark this as an unsolved thread?

Reply:
For me it is fine to move ahead. I take what I can get.